### PR TITLE
feat(phpstan) created a .neon config file to include PHPStan custom rules

### PR DESCRIPTION
**Pull request by dmyios**
## Description

Created phpstan.core.neon cofig file to include PHPStan custom rules from centreon-test-lib. 
Specified the following parameters:
- included .neon file from centreon-test-lib
- specified path to be scanned : src/Core
- specified max level of checks

**Fixes** # MON-14309

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x
- [x] 22.10.x (master)

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: phpstan.core.neon [hunk 1]
```diff
@@ -0,0 +1,7 @@
+includes:
+  - %currentWorkingDirectory%/vendor/centreon/centreon-test-lib/src/PHPStan/phpstan.neon
```
Comment (kduret) [inline phpstan.core.neon]: according to [documentation](https://phpstan.org/config-reference#multiple-files) : `Relative paths in the includes section are resolved based on the directory of the config file is in`
```suggestion
  - vendor/centreon/centreon-test-lib/src/PHPStan/phpstan.neon
```

File: phpstan.core.neon [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+parameters:
+    level: max
+    paths:
+        - src/Core
```
Comment (kduret) [inline phpstan.core.neon:7]: Linked unit tests should also be validated
```suggestion
        - src/Core
        - tests/php/Core
```

Comment (dmyios) [reply phpstan.core.neon:7]: Added tests/php/Core path to .neon

File: phpstan.core.neon (hunk 1 above)
Comment (dmyios) [reply phpstan.core.neon]: Corrected include param

**Review by kduret — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

**Pull request merged**